packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -202,25 +202,33 @@ export class GuxTimeZonePickerBeta {
     });
   }
 
-  private renderDefaultsList(): JSX.Element {
+  private renderDefaultsList(): JSX.Element | null {
```

Comment:
For some reason having a type of JSX.Element or null feels off to me here. I feel like it should be only one type. Is there a way to render an empty JSX.Element maybe instead of null?